A2-service-configs-in-dns.md

```diff
@@ -55,9 +55,12 @@ determine which choice will be selected by a given client:
     // "python", etc).  Each string is case insensitive.
     "clientLanguage": [string],
     // Percentage: integer from 0 to 100 indicating the percentage of
-    // clients that should use this choice.  If present, the number must
-    // match the regular expression `^0|[0-9]|[1-9][0-9]|100$`
-    // All other numbers are considered invalid.
+    // clients that should use this choice.  Each client's resolver should
+    // compute its percentage at startup and use the same percentage for the
+    // lifetime of the resolver; if the client's computed percentage is less
```

Comment:
to be clear, the number picked by the client is [0,100] ?  [0,100) ?  (0,100] ?

Reply:
Answering my own question (and I think you should include in the text:   The random number the client picks should be [0, 100).  The config should apply if the client pct is >= svcConfig.pct.

Reply:
sgtm

Reply:
Neither 0 nor 100 seem particularly useful values for people to use in practice, although I can't think of a reason to reject either of them, since it's clear how to handle both.  What's the advantage in rejecting 100?  Why not reject 0 also?

Reply:
AFAICT @carl-mastrangelo was talking about the random number range used by the client to roll the die, to which I would agree [0, 100) is proper with `client pct < svcConfig.pct` as the logic.  I don't think anyone was arguing 0 or 100 should be disallowed as a setting in the config.

FWIW 0 is useful to disable a config without needing to delete it.  For 100, it's easy enough to delete the percentage field from your config list instead, but I don't see any reason to disallow it, either.

Reply:
Ah, sorry, I misunderstood.  I agree that the number picked by the client should be in the range [0, 100).

I've added a note about this.